src/test/java/com/google/api/generator/spring/composer/goldens/EchoSpringAutoConfigurationFull.golden

```diff
@@ -86,7 +94,7 @@ public class EchoSpringAutoConfiguration {
   @Bean
   @ConditionalOnMissingBean
   public TransportChannelProvider defaultEchoTransportChannelProvider() {
-    return EchoSettings.defaultTransportChannelProvider();
+    return defaultTransportChannelProvider();
```

Comment:
Is this change intentional? (looks like a self-reference now)

Reply:
Ah, it should be getting `EchoSettings` from `types` instead of `STATIC_TYPES` since it's a `VaporReference`. I'm pushing a fix